### ISSUE TITLE
Unexpected CrioArray.set behavior when using with webpack

### ISSUE THREAD

**Issue by nilscox** [planttheidea/crio#53]
Hi!

I'm having this behavior I don't quite understand.

### Steps to reproduce

- `yarn init`
- `yarn add crio webpack webpack-cli`
- copy the code below
- `yarn webpack`
- open the `index.html` in a web browser

### Code

index.js

```js
import crio from 'crio';

const array = crio(['foo', 'bar']);

console.log(array);
console.log(array.set(1, 'cox'));
```

webpack.config.js

```js
module.exports = {
  mode: process.env.NODE_ENV || 'development',
  entry: './index.js',
  output: { filename: 'bundle.js' },
};
```

index.html

```html
<!DOCTYPE html>
<html>
  <head></head>
  <body><script type="text/javascript" src="dist/bundle.js"></script></body>
</html>
```

### Expected output

```
["foo", "bar"]
["foo", "cox"]
```

### Actual output

```
["foo", "bar"]
[null, "cox"]
```

### Extra information

This code works as expected when I don't use webpack (load crio in the HTML and replace the bundle with the code from `index.js` using `const crio = window.crio.default`).

| software | version |
|-|-|
| node | 8.11.1 |
| webpack | 4.15.1 |
| webpack-cli | 3.0.8 |
| crio | 5.0.1 |

**Comment by planttheidea** [comment]
Interesting ... I'll take a look and see if I can reproduce.

**Comment by planttheidea** [comment]
Turned out this was an issue with an underlying dependency, `unchanged` ... luckily that is also my package :smile: so that was fixed quickly. I'll be releasing a new version shortly with the dependency updates which should let this work.

**Comment by planttheidea** [comment]
Released as `5.0.2`. Thanks for calling this out, if you notice anything else let me know!

**Comment by nilscox** [comment]
This works now as expected! 
Thank you for your quick answer, and for this very cool library. :smiley_cat:
